src/blocks/tea_timer.rs

```diff
@@ -0,0 +1,86 @@
+//! Timer
+//!
+//! # Configuration
+//!
+//! Key | Values | Default
+//! ----|--------|--------
+//! `format` | A string to customise the output of this block. See below for available placeholders. | `" $icon $minutes:$seconds "`
+//! `increment` | The numbers of seconds to add each time the block is clicked. | 30
+//!
+//! Placeholder      | Value                                                          | Type   | Unit
+//! -----------------|----------------------------------------------------------------|--------|---------------
+//! `icon`           | A static icon                                                  | Icon   | -
+//! `hours`          | The hours remaining on the timer                               | Text   | h
+//! `minutes`        | The minutes remaining on the timer                             | Text   | mn
+//! `seconds`        | The seconds remaining on the timer                             | Text   | s
+//!
+//! # Example
+//!
+//! ```toml
+//! [[block]]
+//! block = "tea_timer"
+//! format = " $icon $minutes:$seconds "
+//! ```
+//!
+//! # Icons Used
+//! - `tea`
+
+use super::prelude::*;
+use chrono::{Duration, Utc};
+
+#[derive(Deserialize, Debug, SmartDefault)]
+#[serde(deny_unknown_fields, default)]
+pub struct Config {
+    format: FormatConfig,
+    increment: Option<i64>,
+}
+
+pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
+    let interval: Seconds = 1.into();
+    let mut timer = interval.timer();
+
+    let format = config.format.with_default(" $icon {$minutes:$seconds|} ")?;
```

Comment:
To avoid double spaces:
```suggestion
    let format = config.format.with_default(" $icon {$minutes:$seconds |}")?;
```